common/helpers/field/get-field-error-message.js

```diff
@@ -13,12 +15,13 @@ function getFieldErrorMessage({
     return message
   }
 
-  const errorLabel = i18n.t(`fields::${fieldKey}.label`, {
+  const labelFallback = description || question
+  const errorLabel = i18n.t([`fields::${fieldKey}.label`, labelFallback], {
     context: 'with_error',
   })
   const fallback = i18n.t(`validation::${errorType}`, {
     context: 'with_label',
-    label: errorLabel.toLowerCase(),
```

Comment:
Are there any repercussions from removing the `toLowerCase` method ?

Reply:
So I think the slight side-effect might now be `Enter Name of court` instead of `Enter name of court`, however, the lowercasing was having wider side-effects such as `Enter police national computer (pnc)` where we were lowercasing everything.

So to also not mess with the casing of labels coming from the framework I decided to remove it for now. We have the ability to tinker with error messages within the locales but I think that should be a separate task.